Tests/ForageSDKTests/ForageSentryTest.swift

```diff
@@ -0,0 +1,51 @@
+
+import XCTest
+@testable import ForageSDK
+
+struct Event {
+    let message: String
+}
+
+protocol SentryClient {
+    func capture(event: Event)
+}
+class MockSentryClient: SentryClient {
+    var capturedEvents: [Event] = []
+    
+    func capture(event: Event) {
+        capturedEvents.append(event)
+    }
+}
+
+class EventLogger {
+    private let sentryClient: SentryClient
+    
+    init(sentryClient: SentryClient) {
+        self.sentryClient = sentryClient
+    }
+    
+    func logEvent(message: String) {
+        let event = Event(message: message)
+        sentryClient.capture(event: event)
+    }
+}
```

Comment:
We can remove the test changes for this PR

CC from Slack:

your tests seem to be testing the mock sentry client that you've added and not the actual call to SentrySDK.start

Given it's tricky to mock and inject `SentrySDK.start` as a dependency, I don't think you have to add tests^
However, we don't want to initialize `SentrySDK` when running unit tests. Can you make sure that we're not initializing Sentry when running tests?

If Sentry is being initialized while running unit tests, can you pls ask the symphony team how to add guards in our code to prevent Sentry from running in unit tests?

e.g. `SentrySDK.capture` shouldn't work when running unit test.

BTW, Sentry.capture calls will be made from the new ForageLogger (https://github.com/teamforage/forage-ios-sdk/blob/ffdea98330147a177900058ce0fdd206dc12413a/Sources/ForageSDK/Foundation/Telemetry/ForageLogger.swift)

Reply:
af030bf2b15cb9d4019613f73eb1935005424920

Reply:
Thanks for the added context, Danilo! Couldn't seem to find the convo in Slack so am assuming it was a DM. Thanks either way